Fix a few typos in the README files (#173)
near-examples/NFT: README-Windows.md

Testing this contract
=====================

> We have some tests that can be run. The following will just run our simple tests to verify that our contract code is working.

Reviewer: ```suggestion
We have some tests that you can run. For example, the following will run our simple tests to verify that our contract code is working.```
